versioning/README.md

```diff
@@ -5,6 +5,7 @@ Soon, we'll get a separate GitHub Organization. Pretend all this text refers to
 
 Vote on the name by incrementing the tally!
 - [ ] MolSciSchema
+- [ ] MolecularScienceSchema
```

Comment:
How about just `MolecularSchema`?

Reply:
Think we need `Comp` or `Sim` or `Quantum` or `Software` in there? "DielsAlder" and "RareEarth" are not our domain, to my opinion.

Reply:
Going to pull this in. Feel free to add a PR with more suggestions.